### PR TITLE
calamari:add the python rbd api for calamari

### PR DESCRIPTION
**Pull request by songbaisen**
calamari:add the python rbd api for calamari

Signed-off-by: song baisen song.baisen@zte.com.cn

### CODE REVIEW

**Comment by GregMeno** [comment]
@songbaisen This looks awesome! I'd like to merge in 1.3 as master will soon become the same as 1.4.
I'll review and merge it there. Thank you!

**Comment by GregMeno** [comment]
In 1.4 this code would live in calamari-common/calamari_common/remote/mon_remote.py

**Comment by songbaisen** [comment]
@GregMeno  :smile: Thank you sir! I am glad to contribute to calamari.

File: salt/srv/salt/_modules/rbd_api.py [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+class RbdApi():
+
+    def __init__(self, cluster_name="ceph"):
+        self._cluster_name = cluster_name
+
+    def create_image(self, arg_dict):
+        pool_name = arg_dict['pool_name']
+        image_name = arg_dict['image_name']
+        image_size = arg_dict['size']
```
Comment (syf-zsxm) [inline salt/srv/salt/_modules/rbd_api.py]: Can we delete these two lines?

**Comment by songbaisen** [comment]
@GregMeno  Good morning sir! I have restructured the code here.But i am not sure whether the new version code is fit to your project. If it the new version code do not suit, I can roll back to the old version one.Thank you !

**Pull request merged**